src/server/controllers/answers.js

```diff
@@ -3,42 +3,57 @@ import baseController from '~/server/controllers';
 import {
   AnswerBelongsToArtwork,
   AnswerBelongsToProperty,
+  AnswerBelongsToQuestion,
 } from '~/server/database/associations';
 
 import { answerFields } from '~/server/database/associations';
 
 const options = {
   model: Answer,
-  fields: [...answerFields, 'artwork', 'property'],
+  fields: [...answerFields, 'artwork', 'property', 'question'],
   fieldsProtected: ['chainId'],
 };
 
+const readOptions = {
+  ...options,
+
+  include: [
+    AnswerBelongsToProperty,
+    AnswerBelongsToArtwork,
+    AnswerBelongsToQuestion,
+  ],
+  associations: [
+    {
+      association: AnswerBelongsToProperty,
+      destroyCascade: false,
+      fields: ['title'],
+    },
+    {
+      association: AnswerBelongsToArtwork,
+      destroyCascade: false,
+      fields: ['title'],
+    },
+    {
+      association: AnswerBelongsToQuestion,
+      destroyCascade: false,
+      fields: ['title', 'chainId'],
```

Comment:
'chainId' on the question should probably be in fieldsProtected, so that it will be filtered out if this endpoint is ever used without auth